offer-manager-domain/src/main/groovy/life/qbic/business/offers/Offer.groovy

```diff
@@ -331,7 +331,7 @@ class Offer {
      */
     double getTaxCosts() {
         if (!selectedCustomerAffiliation.category.equals(noVatCategory) && selectedCustomerAffiliation.country.equals(countryWithVat)) {
-            return (calculateNetPrice() + getOverheadSum()) * VAT
+            return (calculateNetPrice() + getOverheadSum() - getTotalDiscountAmount()) * VAT
```

Comment:
I think it would be another idea to change `calculateNetPrice()` as it does not calculate the net price right now.

Reply:
Good point. But this would be affecting the other prices too. I expect there some changes soon so I would opt for wait for the input of the stakeholders before refactoring it. But thanks for pointing this out :)